src/arrayMethodJoin.js

```diff
@@ -7,24 +7,16 @@ function applyCustomJoin() {
   [].__proto__.join2 = function(separator = ',') {
     let res = '';
```

Comment:
You should add full names for your values, it's helping your code to be understandable.